### PR TITLE
Use Swift's Syntactic Sugar

### PR DESCRIPTION
**Pull request by wongzigii**
Use the shortcut versions of type declarations over the full generics syntax.

### CODE REVIEW

**Comment by xuyecan** [comment]
Nice advice, but I notice that your commit have some code broke the previous code style. Maybe you should check it again.

File: HandyJSON/HelpingMapper.swift [hunk 1]
```diff
@@ -21,7 +21,7 @@ import Foundation
 
 public class HelpingMapper {
 
-    private var mapping = Dictionary<Int, (String?, ((String) -> ())?)>()
+	private var mapping = [Int: (String?, ((String) -> ())?)]()
```
Comment (xuyecan) [inline HandyJSON/HelpingMapper.swift]: We use 4 whitespaces indentation.

Comment (wongzigii) [inline HandyJSON/HelpingMapper.swift]: Cool, will fix this later.

**Comment by wongzigii** [comment]
Rebased.

**Pull request merged**